src/wallet/rpcwallet.cpp

```diff
@@ -2880,6 +2880,9 @@ static RPCHelpMan listunspent()
                             {RPCResult::Type::STR, "scriptPubKey", "the script key"},
                             {RPCResult::Type::STR_AMOUNT, "amount", "the transaction output amount in " + CURRENCY_UNIT},
                             {RPCResult::Type::NUM, "confirmations", "The number of confirmations"},
+                            {RPCResult::Type::NUM, "ancestorcount", "If transaction is in the mempool, the number of in-mempool ancestor transactions (including this one)"},
```

Comment:
nit: I think there is a slight convention in the existing rpc help to append the conditional at the end, like: "This thing shows XYZ (if available)" (I grepped for available). So I would slightly prefer this and the following to be rewritten something like this:

```
"The number of in-mempool ancestor transactions, including this one (if the transaction is in the mempool)"
```